.ci/preInstall.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# download the HERE SDK
+wget "$HERE_SDK_URL" -q -O HERE_SDK.zip
+unzip -j -d 'HERE_SDK' -o HERE_SDK.zip
```

Comment:
You're mixing single and double quotes in this file with no apparent scheme. Mind either using double quotes everywhere, or single quotes only where no variable interpolation is required?